resources/js/tools.js

```diff
@@ -0,0 +1,7 @@
+function createDict(keys, values) {
```

Comment:
> Is createDict function missing ?

Is it better in another file like that ?

Reply:
Yes, we will probably have more code in common to contentManager and settings